### DISCUSSION TITLE
Trakt Replacement?

### DISCUSSION THREAD

**Discussion by essteewhy** [sbondCo/Watcharr#516] [Q&A]
Ran across Watcharr in the selfhosted reddit.. looks good, is this a Trakt replacement, or can it be?  I'd like something i can backup that saves all my watch history and syncs it back to and from plex.  I'm still using the old Plex Trakt plug-in to do this, but feel any day that'll quit working as its already out of support.   thanks!

**Comment by IRHM** [comment]
Hey @essteewhy, I suppose it could be a trakt replacement, I haven't really used it before so I'm not 100% sure if it can replace it all all aspects, but unfortunately we don't have a two way sync with Plex.

We only have one way sync from Plex to Watcharr, it currently can't go the other way. It's also only a manual sync to Watcharr (via a button in your profile) at the moment, though hopefully we can make this an automatic sync every few hours or so in the near future.

I don't use Plex and I'm not sure if i'll have the time to work on this for a while, though i'd be happy to look at a PR if someone else got it working.

Thanks for creating this discussion!
